tests/components/sfr_box/test_sensor.py

```diff
@@ -32,8 +31,10 @@ async def test_sensors(
     await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
 
-    device_entry = device_registry.async_get_device({(DOMAIN, "e4:5d:51:00:11:22")})
-    assert device_entry == snapshot
+    device_entries = dr.async_entries_for_config_entry(
+        device_registry, config_entry.entry_id
+    )
+    assert device_entries == snapshot
```

Comment:
Small tweak here to align with the other platforms, to check all domain devices not just the expected one.